lib/fastlane/plugin/synx/actions/synx_action.rb

```diff
@@ -12,7 +12,10 @@ def self.run(params)
         cmd << ["--no-default-exclusions"] if params[:no_default_exclusions]
         cmd << ["--no-sort-by-name"] if params[:no_sort_by_name]
         cmd << ["--quiet"] if params[:quiet]
-        cmd << ["--exclusion #{params[:exclusion]}"] if params[:exclusion]
+        if params[:exclusion]
+          Array(params[:exclusion]).each {|exclusion|
+            cmd << ["--exclusion #{exclusion}"] }
```

Comment:
cool! it looks neat! 
Just a final remark, can you replace the curly braces `{}` with `do...end` on the each loop? Just to keep the [ruby code style conventions](https://github.com/bbatsov/ruby-style-guide#single-line-blocks).